### DISCUSSION TITLE
The button cannot be used after RetroArch chooses to restore the default configuration in gba

### DISCUSSION THREAD

**Discussion by lunarstory** [OnionUI/Onion#1629] [General]
The button cannot be used after RetroArch chooses to restore the default configuration in gba

**Comment by Aemiii91** [comment]
Reset RetroArch configuration in **Apps -> Tweaks -> Advanced -> Reset settings -> Reset RetroArch main configuration**.

And maybe **Reset all RetroArch core overrides** as well.
